.github/guides/STANDARDS.md

```diff
@@ -23,11 +23,18 @@ You can avoid hacky code by using object-oriented methodologies, such as overrid
 
 ### Develop Secure Code
 
-* Player input must always be escaped safely, we recommend you use stripped_input in all cases where you would use input. Essentially, just always treat input from players as inherently malicious and design with that use case in mind
+* Player input must always be escaped safely, we recommend you use stripped_input in all cases where you would use input. Essentially, just always treat input from players as inherently malicious and design with that use case in mind.
```

Comment:
Wouldn't `tgui_input_text` be a better default, or worth mentioning? It has `encode = TRUE` by default, which is effectively the same as `stripped_input`.

However, it's worth also mentioning the fact that double-encodes can occur as a result and should be handled to avoid UX issues - albeit that could perhaps be in a stylistic section rather than the security section

Reply:
I chose to leave it alone (the only change made was adding punctuation) but yes the terminology it uses is rather dated as raw `input` isn't widely used anymore